Jenkinsfile

```diff
@@ -6,6 +6,7 @@ pipeline {
   options {
     timestamps()
     buildDiscarder(logRotator(numToKeepStr: '30'))
+    lock resource: "tas-infra"
```

Comment:
Is this to avoid overuse of the Tanzu self-service ?

Reply:
Yep, we can add multiple resources in future, but keeping it serial for now.